README.md

```diff
@@ -33,6 +33,8 @@ https://user-images.githubusercontent.com/44473782/160729779-41fe207c-b5aa-4fed-
 ## Table of Contents
 
 - [Fixing errors](#Fixing-errors)
+- [Others]
```

Comment:
The others section should be at the bottom of the table of contents, but above contribution guidelines